Include changelog for sampling and optimisation initialisation PR (#1394)
pints-team/pints: CHANGELOG.md

### Added
- [#1383](https://github.com/pints-team/pints/pull/1383) Added a method `toy.TwistedGaussianDistribution.untwist` that turns samples from this distribution into samples from a multivariate Gaussian.

> - [#1322](https://github.com/pints-team/pints/pull/1322) Added a method `sample_initial_points` that allows users to generate random initial points with finite metrics (either log-probabilities or error measures) and, if specified, ensures that the points are within the boundaries. The randomness in the initial points can be determined by a user-supplied stochastic function. When a stochastic function is not supplied, if the metric is a log-posterior, the prior will automatically be sampled from; finally, if a stochastic function is not supplied and if the metric is not a log-posterior, if boundaries are supplied, then initial points will automatically be generated from within these.

Reviewer: ```suggestion
- [#1322](https://github.com/pints-team/pints/pull/1322) Added a method `sample_initial_points` that allows users to generate random points with finite metrics (either log-probabilities or error measures) to use as starting points for sampling or optimisation.
```
